fdbt-aws/sql/l.groupOfProducts.sql

```diff
@@ -0,0 +1,14 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+
+DROP TABLE IF EXISTS groupOfProducts;
+
```

Comment:
We need to grant some permissions in the fdbt-aws/sql/z.createUsers.sql file, for this table